packages/flutter/test/material/drawer_test.dart

```diff
@@ -259,6 +259,60 @@ void main() {
     expect(state.isEndDrawerOpen, equals(false));
   });
 
+  testWidgets('Open/close drawer by dragging', (WidgetTester tester) async {
+    final ThemeData draggable = ThemeData(platform: TargetPlatform.android);
```

Comment:
Is setting the platform explicitly needed? (Default platform for test is already Android).

Reply:
If a desktop platform is being targeted, the drawer can't be opened with a swipe gesture.

Even if it's not explicitly necessary, IMO it's probably a good idea to keep it, in the spirit of "self-documenting code".